scripts/presubmit-tests.sh

```diff
@@ -270,7 +270,7 @@ function default_integration_test_runner() {
   local failed=0
   for e2e_test in $(find test/ -name e2e-*tests.sh); do
     echo "Running integration test ${e2e_test}"
-    if ! ${e2e_test} ${options}; then
+    if ! ${e2e_test} "${options}"; then
```

Comment:
I think this will break something; I think $options is intended to be split

Reply:
I have removed the quotes. 
However, `$options` here is always empty as we have `local options=""` at the beginning... I added a comment there and maybe we can remove it in the future.